apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/ResultFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,26 @@
+package ch.tutteli.atrium.api.fluent.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.feature1
+import ch.tutteli.atrium.specs.fun2
+import ch.tutteli.atrium.specs.integration.ResultFeatureAssertionsSpec
+import ch.tutteli.atrium.specs.notImplemented
+
+class ResultFeatureAssertionsSpec : ResultFeatureAssertionsSpec(
+
+
+    feature1<Result<Int>, Int, Int>(Expect<Result<Int>>::isSuccess),
+    fun2<Result<Int>, Int, Expect<Int>.() -> Unit>(Expect<Result<Int>>::isSuccess)
```

Comment:
similar you have to use `fun1` here as it expects only the `Expect<Int>.() -> Unit`